app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -3024,8 +3081,9 @@ public function getScopeByMailChimpStoreId($mailChimpStoreId)
         $collection = Mage::getResourceModel('core/config_data_collection')
             ->addFieldToFilter('path', array('eq' => Ebizmarts_MailChimp_Model_Config::GENERAL_MCSTOREID))
             ->addFieldToFilter('value', array('eq' => $mailChimpStoreId));
+
         if ($collection->getSize()) {
-            $configEntry = $collection->getFirstItem();
+            $configEntry = $collection->setPageSize(1, 1)->getLastItem();
```

Comment:
Move setPageSize filter to the collection definition to keep everything together.
setPageSize function only takes 1 parameter.

Reply:
Solved